### ISSUE TITLE
imageButton.searchButton  Does not Have an Accessible Name

### ISSUE THREAD

**Issue by LanieOkorodudu** [UniversalViewer/universalviewer#1100]
UV version:
 universalviewer@<4.0.25>

I'm submitting a:
 bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below

Current behaviour:
The button with the class `imageButton searchButton` is missing an accessible name and does not provide a description of its functionality.

Expected behaviour:
The button should have an accessible name that provides a clear description of its purpose. For example, it could be displayed as "Search Button" if it initiates a search action.

Steps to reproduce:
1. Navigate [here](https://www.universalviewer.dev/#?xywh=327%2C581%2C3184%2C1898).
2. Hover the mouse over the imageButton searchButton.
![Screenshot 2024-09-04 141656](https://github.com/user-attachments/assets/09dbc155-7849-463b-929a-847abe17f675)
3. Observe that no tooltip or description of the button’s functionality appears

Proposed Solution:
Add an `aria-label` attribute or use the `title` attribute to provide a meaningful name.

Other information:
This issue fails to meet [WCAG 2.1 Success Criterion 4.1.2](https://www.wcag.com/developers/4-1-2-name-role-value/), which requires that interactive elements, like buttons, must have accessible names.

Related code:
```html
<button class="imageButton searchButton"></button>
